### PR TITLE
Register ControlCore [https://github.com/KTH-AC/ControlCore.jl.git]: …

### PR DESCRIPTION
**Pull request by aytekinar**
…v0.0.1

### CODE REVIEW

File: ControlCore/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline ControlCore/versions/0.0.1/requires]: should really declare a minimum version dependency that implements the features that you need

Comment (aytekinar) [inline ControlCore/versions/0.0.1/requires]: Sorry. We have missed this thing. It is a WIP now --- we are doing some code enhancements and we are trying to get around `Julia v0.5.0` failing tests. However, we need this package for another one that depends on it.

Thanks for the warning!

File: ControlCore/versions/0.0.1/sha1 [hunk 2]
```diff
@@ -0,0 +1 @@
+064d9e09397a15a2e52e8fed8460527b34da0687
```
Comment (tkelman) [inline ControlCore/versions/0.0.1/sha1]: the commit should match https://github.com/KTH-AC/ControlCore.jl/releases

Comment (aytekinar) [inline ControlCore/versions/0.0.1/sha1]: May I get the commit hash directly from the repo and paste into our METADATA? I created a signed commit; but obviously it changes the hash. Or do you strictly require that we use `PkgDev` at all times?

Comment (tkelman) [inline ControlCore/versions/0.0.1/sha1]: PkgDev is recommended, if it's working for you. You can just modify the contents of this file in this branch of your metadata fork though, if you need to make modifications

Comment (aytekinar) [inline ControlCore/versions/0.0.1/sha1]: I think I have done it correctly this time. Changes:
1.  I had fixed a typo in the package home, and I have included this one in the version tag,
2.  Tag is signed,
3.  `PkgDev.register()` is used,
4.  Our fork's branch is rebased.

Comment (tkelman) [inline ControlCore/versions/0.0.1/sha1]: Looks right. 2 is optional right now, but doesn't hurt anything.

**Comment by tkelman** [comment]
See also https://github.com/JuliaLang/METADATA.jl/pull/6454#issuecomment-249055394

**Pull request merged**